src/styles/product__card.scss

```diff
@@ -0,0 +1,121 @@
+$gap-between-cards: 48px;
+$product-card-width: 200px;
+$card-number-in-row: 4;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
move in the * selector

Reply:
still not fixed
https://github.com/mate-academy/layout_catalog/pull/2744/files#r1119669010

Reply:
* not fixed, move box-sizing to * selector and remove from here